tests/gocase/unit/type/json/json_test.go

```diff
@@ -612,6 +612,19 @@ func TestJson(t *testing.T) {
 		require.EqualValues(t, "[]", vals[1])
 
 	})
+
+	t.Run("JSON.MSET basics", func(t *testing.T) {
+		require.Error(t, rdb.Do(ctx, "JSON.MSET", "a0", "$.a", `{"a": 1, "b": 2, "nested": {"a": 3}, "c": null}`, "a1", "$", `{"a": 4, "b": 5, "nested": {"a": 6}, "c": null}`).Err())
```

Comment:
Maybe we should delete `a0` first to avoid being interference by other test cases.